src/core/ui.js

```diff
@@ -114,6 +114,58 @@ function createWarnButton(butName, arr, title) {
     ["label", `Document ${title.toLowerCase()}`],
   ]);
   ariaDecorate(button, ariaMap);
+
+
+  var numSteps = 20.0;
```

Comment:
Using `const` instead of `var` would be better for all the variables here, except `boxElement` there too `let` could be used.
Also this `numSteps` is redundant.